index.html

```diff
@@ -1703,12 +1713,18 @@
 			'mapnum': {},
 			'order': priority_match([modules['I']['order'], modules['II']['order'], modules['III']['order']]),
 			'money': priority_match([modules['I']['money'], modules['II']['money'], modules['III']['money']]),
+			// create tokens
+			getNameTokens: getNameTokens,
```

Comment:
I was having a lot of issues getting this function call to work properly (I am not a js programmer) and this seemed to be best practice. that is the only place it is called so if this isn't necessary I'm happy for it to be removed.